### PR TITLE
added new ordinals and tms yaml files

### LINKED ISSUE #314: Add Ordinals API docs
As part of the "References" dropdown, add "Ordinals API" as the 2nd option, and next to "Stacks Blockchain API". The OpenAPI YAML is available at: https://ordinals-api.vercel.app/openapi.yaml.

The desired docs path would be: https://docs.hiro.so/ordinals.

As part of this effort, let's also cleanup https://github.com/hirosystems/docs/commit/bbd0d5a3609392f94524b2fdeb397283d943c15b by removing the physical file of TMS with the HTTP resolvable file at https://token-metadata-service.vercel.app/openapi.yaml.

Ensure both the URLs are correctly redirected.

### PR DESCRIPTION
**Pull request by trichards-hiro**
## Description

Added new Ordinals API and TMS API files.

This fixes #314

### CODE REVIEW

**Review by hugocaillard — APPROVED** [review]
Files format looks good to me 👍

**Review by rafaelcr — APPROVED** [review]

**Review by LakshmiLavanyaKasturi — APPROVED** [review]

File: docusaurus.config.js [hunk 1]
```diff
@@ -317,9 +321,13 @@ module.exports = {
...
                 to: '/api',
               },
+              {
+                label: 'Ordinals API',
+                to: '/ordinals',
+              },
               {
                 label: 'Token Metadata Service',
-                to: '/metadata',
+                to: '/token-metadata-service',
```
Comment (sabbyanandan) [inline docusaurus.config.js:330]: This was requested to be at `/metadata`.

File: docusaurus.config.js [hunk 2]
```diff
@@ -279,8 +279,12 @@ module.exports = {
             spec: 'https://raw.githubusercontent.com/hirosystems/stacks-blockchain-api/gh-pages/openapi.resolved.yaml',
           },
           {
-            route: '/metadata/',
-            spec: './docs/tms-openapi.json',
+            route: '/ordinals',
+            spec: './docs/ordinals-openapi.yaml',
```
Comment (sabbyanandan) [inline docusaurus.config.js:283]: This should not be from the physical file, but from the HTTP location as listed in https://github.com/hirosystems/docs/issues/314.

File: docusaurus.config.js [hunk 3]
```diff
@@ -279,8 +279,12 @@ module.exports = {
...
+          },
+          {
+            route: '/token-metadata-service',
+            spec: './docs/tms-openapi.yaml',
```
Comment (sabbyanandan) [inline docusaurus.config.js:287]: Same as previous comment here, too. We need to delete the physical files from the repo and replace it with HTTP URL.

**Review by sabbyanandan — COMMENTED** [review]
A few things are incorrect. Let's please fix them before the annoucement.

File: docusaurus.config.js (hunk 1 above)
Comment (LakshmiLavanyaKasturi) [reply docusaurus.config.js:330]: @sabbyanandan: /metadata throws 404. It works with /token-metadata-service

Comment (hugocaillard) [reply docusaurus.config.js:330]: I had a call with @trichards-hiro  this morning to handle some git/docusaurus issues, I didn't know which was the right URL.

So if you change [here](https://github.com/hirosystems/docs/pull/317/files#diff-a038096cbdea434999e1dce5ab497212f1fe18204dde1a027ce3bdd663261a2aR286) and [here](https://github.com/hirosystems/docs/pull/317/files#diff-a038096cbdea434999e1dce5ab497212f1fe18204dde1a027ce3bdd663261a2aR330), the URL can be whatever we want, it just has to match 👌

**Pull request merged**